src/rgw/rgw_lc_s3.cc

```diff
@@ -11,6 +11,8 @@
 #include "rgw_user.h"
 #include "rgw_lc_s3.h"
 
+static constexpr uint32_t MAX_TAG_KEY_SIZE=128;
```

Comment:
I'm not sure we want to unconditionally enforce this

Reply:
why not, we make some check in case that users reconfigure those tags filter after they found object tags has length limits.